atmo/urls.py

```diff
@@ -17,5 +17,6 @@
     # contribute.json url
     url(r'^(?P<path>contribute\.json)$', static.serve, {'document_root': settings.ROOT}),
 
+    url(r'^django-rq/', include('django_rq.urls')),
```

Comment:
The dashboard [uses the Django admin templates](https://github.com/ui/django-rq/blob/3d0091b362152bbacb23a3a810572eb72f574d9f/django_rq/templates/django_rq/stats.html#L1), so let's mount that under `admin/rq/` instead of `django-rq`